website/static/js/fangorn.js

```diff
@@ -726,7 +726,7 @@ function doItemOp(operation, to, from, rename, conflict) {
         orderFolder.call(tb, from.parent());
     }).always(function(){
 
-        tb.pendingReadyFiles = tb.pendingReadyFiles.filter(function (file) { return file.data.id !== from.data.id; });
+        tb.pendingReadyFiles = [] || tb.pendingReadyFiles.filter(function (file) { return file.data.id !== from.data.id; });
```

Comment:
```suggestion
        tb.pendingReadyFiles = (tb.pendingReadyFiles || []).filter(function (file) { return file.data.id !== from.data.id; });
```
I'm probably missing something??

Reply:
It makes no difference regarding errors thrown

Reply:
hm, this avoids the typeError caused by calling filter on undefined obj. Filtering on an empty array is fine. Also with `[] || tb.pendingReadyFiles.filter(....`,  `tb.pendingReadyFiles.filter(` will never run.